### PR TITLE
Release 12.6.3.1

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
back-ported fix from develop to common/class.Logger.php

### CODE REVIEW

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by siwane — APPROVED** [review]
Legacy code, any chance to have unit test?

- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: manifest.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
     'label' => 'Generis Core',
     'description' => 'Core extension, provide the low level framework and an API to manage ontologies',
     'license' => 'GPL-2.0',
-    'version' => '12.6.3',
+    'version' => '12.6.3.1',
```
Comment (siwane) [inline manifest.php:37]: If possible we should privilege semver usage. As 12.6.4 tag is not used, you should go for 12.6.4

**Review by siwane — CHANGES_REQUESTED** [review]

File: manifest.php (hunk 1 above)
Comment (mike-ionut-mihai-sandu-tao) [reply manifest.php:37]: i wanted to be 100 sure that everyone understands that this is a back port, and that's why a not so proper semver, 
but i will triple check the 12.6.4 tag and update to that
thank you

**Review by siwane — APPROVED** [review]

**Pull request merged**